lib/pipedrive/deals.ex

```diff
@@ -19,6 +19,17 @@ defmodule Pipedrive.Deals do
     API.get("/deals", "", opts)
   end
 
+  @doc """
+  Get a deal by ID.
+
+  [Pipedrive API docs](#{api_docs_base_url()}/Deals/get_deals_id)
+  """
+  @impl Pipedrive.RESTEntity
+  @spec get(String.t(), Keyword.t()) :: API.response()
+  def get(id, opts \\ []) do
```

Comment:
Is getting by id necessary because the results of search do not include all necessary details?

Reply:
Jawohl https://github.com/steady-media/main_app/blob/f91e915f1b73f08b36a641e3f81999b5ce644055/lib/ypsilon/pipedrive/person.ex#L42-L44